presto/presto.sh

```diff
@@ -215,11 +206,9 @@ EOF
 }
 
 function configure_and_start_presto(){
-  # Copy required Jars
-  cp ${CONNECTOR_JAR} presto-server-${PRESTO_VERSION}/plugin/hive-hadoop2
 
   # Configure Presto
-  mkdir -p presto-server-${PRESTO_VERSION}/etc/catalog
+  mkdir -p presto-server-${STARBURST_PRESTO_VERSION}/etc/catalog
```

Comment:
it's weird place to put presto configuration. Normally you'd use `/etc/presto` & `/etc/presto/catalog`
I think we could
- use standard location for presto configs: `/etc/presto` & `/etc/presto/catalog`
- make `ln -s /etc/presto "presto-server-${STARBURST_PRESTO_VERSION}/etc"` for compatibility with DataProc users already using Presto

Reply:
I agree. I'd rather do this in a second pass when we improve the script. I would prefer to leave as is for now as the goal was to basically leverage what exists and put in Starburst.